vscode-extensions/inscription/webview/src/index.tsx

```diff
@@ -78,13 +79,21 @@ function startInscriptionClient(webSocketAddress: string) {
 }
 
 export function render(inscriptionClient: InscriptionClient, selectedElement: SelectedElement = undefined): React.ReactElement {
+  const queryClient = new QueryClient();
```

Comment:
It seems to be complex to integrate the inscription view app now. We need to know stuff about the react-query. 
I would like to see such things hidden in the inscription view app itself. 
Also the configuration with the three different parameters could be simplified by providing the selectedElement in one arg directly.

Reply:
good ideas

Reply:
I agree, that this it is not ideal that the vscode extension needs to know this stuff here! 
We definitly should move this more into the inscription view code, so this stuff here is not needed.
I wasn't aware of this, sorry.